src/analytics/Events.tsx

```diff
@@ -677,4 +677,5 @@ export enum PointsEvents {
 
 export enum EarnEvents {
   earn_cta_press = 'earn_cta_press',
+  earn_tap_add_crypto_action = 'earn_tap_add_crypto_action',
```

Comment:
```suggestion
  earn_add_crypto_action_press = 'earn_add_crypto_action_press',
```
to be consistent with existing earn event